docs/dashboard/dashboard.rst

```diff
@@ -0,0 +1,275 @@
+The project overview
+--------------------
+
+When opening the dashboard the project overview page will be the first page greeting you.
+It has several elements:
+
+.. figure:: /dashboard/images/img_1.png
+   :width: 100%
+   :align: center
+   :alt: The project overview
+
+   The project overview page
+
+Let's go over what some of these buttons do:
+
+1. The inmanta "home" button: takes you back to this page.
+2. List of created projects: Lists all existing projects.
+   Clicking on the name of a project will take you to its page.
+3. Project delete button:
+   Deletes the project and all the environments it contains.
+   This will delete the history and environments, but it will not purge the system of changes made or managed by inmanta.
+4. Report an issue:
+   If you run into any issues/bugs, this button will take you to github with the "new issue" page open for the inmanta repository.
+
+.. figure:: /dashboard/images/img_2.png
+   :width: 100%
+   :align: center
+   :alt: The project overview
+
+   The project overview page
+
+5. Environment navigation button:
+   Displays a list of projects and their environments.
+   Allows for navigate to any environment, by simply clicking it's name.
+6. Create new project button:
+   This will take you through the creation of a new project and the creation of its first environment.
+7. Green checkmark:
+   This will take you to the orchestrator status page, displaying all sorts of useful information about the orchestrator instance.
+   If the dashboard loses its connection to the server, this green checkmark will turn into a red cross.
+
+Create a new project
+--------------------
+
+Using the ``Add new project`` button we can create new projects:
+
+.. figure:: /dashboard/images/img_3.png
+   :width: 100%
+   :align: center
+   :alt: Adding a new project
+
+   Adding a new project
+
+Once create is pressed, you are immediately taken to the "Create a new Environment" screen.
+This will help you set up your first environment.
+Pressing cancel will leave the project empty.
+
+.. figure:: /dashboard/images/img_4.png
+   :width: 100%
+   :align: center
+   :alt: Creating a new Environment
+
+   Creating a new Environment
+
+The two screenshots above are equivalent to the following inmanta-cli commands:
+
+.. code-block:: sh
+  :linenos:
+
+  inmanta-cli project create -n dashboard-test
+  inmanta-cli environment create -n quickstart-env -p quickstart -r https://github.com/inmanta/quickstart.git -b master
+
+When in an environment, a new button at the bottom will appear:
+
+.. figure:: /dashboard/images/img_17.png
+   :width: 100%
+   :align: center
+   :alt: The stop button
+
+   The emergency stop button
+
+This big red button will stop any and all inmanta operations for the current environment.
```

Comment:
I think this could be misconstrued as stopping the resources themselves? Deployed resources will not be removed, so their "operations" will keep going. I think people might understand it that way if we don't leave inmanta in.

Reply:
replaced "any and all" with "any"